src/core/trulens/core/database/migrations/versions/1_first_revision.py

```diff
@@ -33,6 +33,7 @@ def upgrade(config) -> None:
         sa.Column(
             "feedback_definition_id", sa.VARCHAR(length=256), nullable=False
         ),
+        sa.Column("run_location", sa.Text(), nullable=True),
```

Comment:
General question for my understanding - how do we name the files 1_first_revision vs 2_second_revision? Also When is this script run?